package-lock.json

```diff
@@ -872,7 +872,7 @@
       "dev": true
     },
     "clinical-trial-matching-service": {
-      "version": "github:mcode/clinical-trial-matching-service#be0a3b60d6837a3b4e68860f4992e6a908c27166",
+      "version": "github:mcode/clinical-trial-matching-service#7a334abb331d5f71da6fdbf594f787a91750926e",
```

Comment:
This appears to be downgrading the service library to an earlier version. There appears to be a weird issue where NPM sometimes just does that on you, I'm not sure why, but it should be changed to use the latest version.